.github/workflows/Android.yml

```diff
@@ -0,0 +1,19 @@
+name: uvwasi Android CI
+
+on: [push, pull_request]
+
+jobs:
+  build-android:
+    runs-on: ubuntu-latest
+    container: reactnativecommunity/react-native-android:2020-5-20
+    steps:
+      - uses: actions/checkout@v2
+      - name: Envinfo
+        run: npx envinfo
+      - name: Build android arm64
+        # see build options you can use in https://developer.android.com/ndk/guides/cmake
+        run: |
+          apt update && apt install python3-pip -y && pip3 install cmake
```

Comment:
These steps look a good bit different from what you did in https://github.com/libuv/libuv/pull/2858. Is there a reason for that?

Reply:
Yeap, the bundled cmake is too old, doesn't support `FetchContent_Declare`.

Reply:
Ah, that makes sense. Thanks.